src/missiondef.cpp

```diff
@@ -583,7 +583,7 @@ MISSION(_("Find Flag"), MGOAL_FIND_ITEM, 2, 100000, false,
          &mission_place::always, &mission_start::ranch_bartender_4,
          &mission_end::standard, &mission_fail::standard);
   ORIGINS(ORIGIN_SECONDARY);
-  ITEM("55gal_drum");
+  ITEM("metal_bigdrum");
```

Comment:
Changed item type ids also change recipe id (they depend on the result id), which is stored in the player activity and therefor in a saves (autosave while crafting).

Recipe ids are used in the `player::learned_recipes`.

Item type ids are stored in`player::items_identified`. 

`Character::assigned_invlet` uses the item id, as does `inventory::invlet_cache`, through invalid item ids there are probably not a real problem.

There is also `npc_favor::item_id`, but that does seem to be completely unused, so you may as well ignored it.

`monster::ammo` contains item type ids.

Neither of those may make any problems with the proposed changes here. But they have the potential to break. Changing the id of a book will suddenly mark is as unread (not in `items_identified`), a recipe might be unlearned (not in `learned_recipes`).